bin/mail/2016-04-26-changeorg-petition/generate-lists.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US contacts
+us_contacts_query = %q(
+  !(role_s: student) && (
+    create_ip_country_s: "United States" ||
+    hoc_country_s: "us" ||
+    location_country_code_s: "US" ||
+    location_country_s: "United States"
```

Comment:
Thanks. I got the initial list from [here](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/mail/mailing-common/mailing-list-utils.rb#L38-L38) and ran a facet query on each one individually to make sure it was present. 

I'll remove location_country_s in favor of location_country_code_s and add a comment linking to the above. Can you think of any other conditions that might be relevant here?